CHANGELOG.asciidoc

```diff
@@ -34,13 +34,18 @@ This will be the last minor release to support the following versions:
  * Django 1.8, 1.9 and 1.10
  * Flask < 1.0
 
+ In addition, as of this release we no longer support positional args for
```

Comment:
Wondering if we should tone down this a little bit, it sounds like things would break if the user uses position args.

Something like

> In addition, as of this release we only supported capturing extended information on elasticsearch queries when using keyword arguments with the elasticsearch-py API. This is in keeping with the
 https://elasticsearch-py.readthedocs.io/en/master/api.html#api-documentation[upstream policy]
 of positional arguments being unsupported. {pull}697[#697]